lib/streamlit/elements/json.py

```diff
@@ -39,26 +37,11 @@
 Body: TypeAlias = Union[List[Any], Tuple[Any, ...], MutableMapping[Hashable, Any]]
 
 
-def _convert_sets_to_lists(body: Body) -> Body:
-    if isinstance(body, (list, tuple)):
-        # We could technically iterate through the elements of a list/tuple and convert
-        # any sets that we find to lists like we do below, but lists/tuples of sets
-        # seem like a strange enough use-case that it's probably not worth the
-        # additional complexity.
-        return body
-
-    # Convert sets into lists, which render more nicely on the frontend
-    set_found = False
-    for key in body:
-        if isinstance(body[key], set):
-            if not set_found:
-                # When set is found to prevent mutation of input body, we need
-                # to shallow copy it once. To avoid copying it multiple times
-                # we use set_found flag.
-                body = copy.copy(body)
-                set_found = True
-            body[key] = list(body[key])
-    return body
+def _convert_sets_to_lists_repr(o: Any) -> Union[str, List[Any]]:
```

Comment:
nit: I like to use `o: object` over `o: Any` whenever I can. That is, when the function should truly handle any object. Makes it easier to not break that contract at a later point.